perf/perf.ts

```diff
@@ -198,8 +197,35 @@ export async function runAll(groups: string[]): Promise<void> {
   }
   const benchmarks = findBenchmarks(groups);
   for (const b of benchmarks) {
-    const r = await runBenchmark(b, 'replicache');
-    out.textContent += r + '\n';
+    try {
+      const result = await runBenchmark(b);
+      if (result) {
+        out.textContent += formatAsReplicache(result) + '\n';
+      }
+    } catch (e) {
+      out.textContent += `${b.name} had an error: ${e}` + '\n';
+    }
   }
   out.textContent += 'Done!\n';
 }
+
+// See https://github.com/benchmark-action/github-action-benchmark#examples
+type Entry = {
+  name: string;
+  unit: string;
+  value: number;
+  // variance
+  range?: string;
+  // any extra info, will be displayed in tool tip on graphs
+  extra?: string;
+};
+
+function createGithubActionBenchmarkJsonEntry(result: BenchmarkResult): Entry {
```

Comment:
Json -> JSON

(I am also having a hard time with this)

Reply:
done